powerhome/playbook
# Fix for incorrect order of classNames fixes #1058

#### Screens

Non-visual change

#### Breaking Changes

Nope

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-142

#### How to test this

This is mainly a dev fix to ensure custom class names that are added to a kit display correctly. Confirm the kits load as expected. The bug was reported in #1058 on the installer schedule, so perhaps you can see the extra padding go away.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
